inputters/pandocast.lua

```diff
@@ -582,9 +582,12 @@ function inputter.appropriate (round, filename, doc)
 end
 
 function inputter:parse (doc)
-  local has_json, json = pcall(require, "json.decode")
+  local has_json, json = pcall(require, "lunajson")
```

Comment:
Would still be best to prioritize luajson (C-based so likely more efficient) over lunajson and use the latter as fallback.
We could then even have it as dependency and simplify everything (one could only go through the burden of installing luajson for performances), and thus close also #21 while having a stock release that always works.